assets/model_monitoring/components/model_monitor/model_monitor_output_metrics/spec.yaml

```diff
@@ -31,6 +31,10 @@ inputs:
     type: mltable
     mode: direct
     optional: true
+  feature_importance_all:
```

Comment:
Can you upgrade the component version? Also, you need to upgrade the signal version for all the signals using this component. You can refer to this PR https://github.com/Azure/azureml-assets/pull/2136/files#diff-9845970428817310efa54b26e7c1456a1d011f8e28327bfc6455120f6167e0d0